### DISCUSSION TITLE
Rendering Slots on the player

### DISCUSSION THREAD

**Discussion by ericon7** [TheIllusiveC4/Curios#374] [Q&A]
Hiya! I cant seem to find any other tutorials, so ill ask here.
How do I get a slot to render on the player? I put a block of glowstone in the charm slot and I wanna see it in game and I am clueless as to how to do it. I also have next to no programming experience/knowledge. sorry.
Would appreciate the help! Thank you!

**Comment by TheIllusiveC4** [comment]
Unfortunately there is currently no way to accomplish this without programming a mod to do so, as the rendering of the item in the slot requires code to be written in order to function properly.

**Reply by ericon7** [reply]
is there any mods i can adapt code from?
or what folder i can look through?

**Reply by TheIllusiveC4** [reply]
One of my other mods, Charm of Undying, has some rendering code for rendering an item:
https://github.com/illusivesoulworks/charmofundying/blob/1.20.2/common/src/main/java/com/illusivesoulworks/charmofundying/client/TotemRenderer.java
But I don't know how useful that will be as it requires some programming knowledge and modding experience to understand.

**Reply by ericon7** [reply]
ok well if i got a baseline i can trial and error. 
thanks man!
